platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -58,8 +58,8 @@ - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
     if (range.location + range.length > [self pointCount])
     {
         [NSException raise:NSRangeException
-                    format:@"Invalid coordinate range %@ extends beyond current coordinate count of %@",
-                        NSStringFromRange(range), @([self pointCount])];
+                    format:@"Invalid coordinate range %@ extends beyond current coordinate count of %ld",
+                        NSStringFromRange(range), (long)[self pointCount]];
```

Comment:
Shouldn't this be unsigned? What I usually do is use `%@` with an uncasted pointCount and wait for an Xcode fix-it to tell me which format specifier and cast to use.